### DISCUSSION TITLE
Section 2.4 Performance, C and Cython

### DISCUSSION THREAD

**Discussion by sergiopasra** [astropy/astropy-v5.0-paper#101] [Text comments] [answered]
In section 2.4, it is mentioned that some code time-critical was written in C. Is it worth including Cython code here also?
In that regard, Cython is cited in page 29 but it is not mentioned anywhere else in the text.

**Comment by adrn** [comment, marked as answer]
Thanks. I addressed this in a commit that I will push shortly.
